force-app/main/default/permissionsets/Skriv_til_oss_Service_Cloud_tilgang.permissionset-meta.xml

```diff
@@ -131,21 +136,44 @@
     </fieldPermissions>
     <fieldPermissions>
         <editable>false</editable>
-        <field>Thread__c.CRM_Theme_Group_Name__c</field>
+        <field>Thread__c.CRM_Theme__c</field>
         <readable>true</readable>
     </fieldPermissions>
     <fieldPermissions>
         <editable>false</editable>
         <field>Thread__c.CRM_isActive__c</field>
         <readable>true</readable>
     </fieldPermissions>
+    <fieldPermissions>
+        <editable>false</editable>
+        <field>Thread__c.STO_Medskriv__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
     <fieldPermissions>
         <editable>false</editable>
         <field>Thread__c.STO_Sensitive_Information__c</field>
         <readable>true</readable>
     </fieldPermissions>
     <hasActivationRequired>false</hasActivationRequired>
     <label>Skriv til oss - Service Cloud tilgang</label>
+    <objectPermissions>
+        <allowCreate>false</allowCreate>
+        <allowDelete>false</allowDelete>
+        <allowEdit>false</allowEdit>
```

Comment:
I hvilket perm set styres edit permissions for Case egentlig? Det er vel det objektet som jobbes mest med? Trodde det var service cloud perm set

Reply:
Tilgang til Case-objektet gis i utgangspunktet via Case Sharing Rule i Sharing Settings. Ser vi har et permission set kalt NKS_Case som gir lese og skrive tilgang til Case-objektet samt at det gis lese-tilgang via Service Cloud også slik du nevnte.